### PR TITLE
SASS-7847: UK Rent a Room: Expenses Section - Create 'Other allowable property expenses' page

### PR DESCRIPTION
**Pull request by willjshark**

### CODE REVIEW

File: app/controllers/OtherPropertyRRExpensesController.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
```
Comment (arunkumar461) [inline app/controllers/OtherPropertyRRExpensesController.scala:1]: format all files using scalafmt

File: conf/messages.en [hunk 2]
```diff
@@ -1672,3 +1671,34 @@ ukrentaroom.legalManagementOtherFee.error.nonNumeric.agent = Enter an amount for
...
+otherPropertyRRExpenses.details.content.l7.individual = training courses related to your business, for example refresher courses
+otherPropertyRRExpenses.details.content.l7.agent = training courses related to their business, for example refresher courses
+otherPropertyRRExpenses.details.link.href = https://www.gov.uk/expenses-if-youre-self-employed
+otherPropertyRRExpenses.details.link.text1 = Read more about
+otherPropertyRRExpenses.details.link.text2 = allowable property expenses at GOV.UK (opens in new tab)
+otherPropertyRRExpenses.error.required.individual = Enter an amount for your other allowable property expenses
+otherPropertyRRExpenses.error.required.agent = Enter an amount for your client's other allowable property expenses
+otherPropertyRRExpenses.error.twoDecimalPlaces.individual = Amount can only include pounds and pence, for example £600 or £600.20
+otherPropertyRRExpenses.error.twoDecimalPlaces.agent = Amount can only include pounds and pence, for example £600 or £600.20
+otherPropertyRRExpenses.error.nonNumeric.individual = Enter an amount for your other allowable property expenses
```
Comment (arunkumar461) [inline conf/messages.en:1701]: nonNumeric & twoDecimalPlaces should have same error messages

File: conf/app.routes [hunk 3]
```diff
@@ -442,3 +442,9 @@ GET        /:taxYear/uk-rent-a-room/expenses/legal-management-other-fee
 POST       /:taxYear/uk-rent-a-room/expenses/legal-management-other-fee                  controllers.ukrentaroom.LegalManagementOtherFeeController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
 GET        /:taxYear/uk-rent-a-room/expenses/change-legal-management-other-fee                        controllers.ukrentaroom.LegalManagementOtherFeeController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
 POST       /:taxYear/uk-rent-a-room/expenses/change-legal-management-other-fee                        controllers.ukrentaroom.LegalManagementOtherFeeController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
+
+GET        /:taxYear/uk-rent-a-room/other-property-rr-expenses                       controllers.OtherPropertyRRExpensesController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
```
Comment (arunkumar461) [inline conf/app.routes:446]: To make it consistent:  /:taxYear/uk-rent-a-room/expenses/other-allowable-property-expenses

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

File: app/controllers/OtherPropertyRRExpensesController.scala (hunk 1 above)
Comment (willjshark) [reply app/controllers/OtherPropertyRRExpensesController.scala:1]: Fixed with latest commit

File: conf/messages.en (hunk 2 above)
Comment (willjshark) [reply conf/messages.en]: Fixed with latest commit

File: conf/app.routes (hunk 3 above)
Comment (willjshark) [reply conf/app.routes]: Fixed with latest commit

**Pull request merged**